### ISSUE TITLE
support operation duration

### ISSUE THREAD

**Issue by gclove** [getsentry/sentry-php#1242]
Can we support operation duration? I think there is such a display in the vue package.

> ![image](https://user-images.githubusercontent.com/3645410/133869436-b38b55e2-b497-4b1a-b8d0-c847ee4e4bf1.png)

> ![image](https://user-images.githubusercontent.com/3645410/133869505-45306cb3-3e3f-4fa4-9bd9-b1e13b872e74.png)

**Comment by ste93cry** [comment]
I don't see why not, but I'm not sure how to do this nor if there is something that we really have to do. Maybe @kamilogorek can shed some light on this?

**Comment by mfb** [comment]
> In the Symfony SDK we [used](https://github.com/getsentry/sentry-symfony/blob/cbd486cdf5b9d1933e0dad1c72a84269397ca931/src/Tracing/Doctrine/DBAL/TracingDriverConnection.php#L24) a different convention because we trace queries at an even deeper level by distinguishing the different phases that are involved (executing the statement, fetching the data, etc)

Hmm might be a good idea to change those from sql.conn.prepare to db.conn.prepare etc.?

**Comment by mfb** [comment]
btw, I noticed Sentry has some updated docs on span operations: https://develop.sentry.dev/sdk/performance/span-operations/

**Comment by cleptric** [comment]
Is there anything left to be done?

**Comment by stayallive** [comment]
Possibly the framework integrations need to update their span names to comply with the doc (https://develop.sentry.dev/sdk/performance/span-operations/) to allow for Sentry server to show info for them. We did this a while back in Laravel (https://github.com/getsentry/sentry-laravel/pull/533) but I'm not sure we ever looked at that for Symfony?

**Comment by cleptric** [comment]
@stayallive Yep, I took a look and we have to fix up some stuff over at Symfony.
